tests/cupyx_tests/scipy_tests/ndimage_tests/test_filters.py

```diff
@@ -238,6 +252,18 @@ def test_filter(self, xp, scp):
             testing.shaped_random((26, 3, 52, 70, 50), xp, xp.float32),
             [1, 1, 1, 1, 1], mode='nearest')
 
+    @testing.with_requires('scipy>=1.x.x')
```

Comment:
Given we don't know the exact SciPy version for now, how about the following?
```python
requires_scipy_ndimage_backend = pytest.mark.skip('scipy.ndimage backend feature has not been released')
# TODO: After the feature is released
# requires_scipy_ndimage_backend = testing.with_requires('scipy>=1.x.x')
...
    @requires_scipy_ndimage_backend
    def test_...
```

Reply:
That sounds good! Thanks